src/data/camos/classic.js

```diff
@@ -0,0 +1,11 @@
+import classic from '../requirements/camo/classic'
+const base = Object.keys(classic)
+
+const dlc = []
```

Comment:
Will there be DLC camouflages? 🤔

Reply:
I would hope so, if they're doing unique camos per weapon it only makes sense they introduce more camos with the new seasons! 😄